Pear/Controller/HomeViewController.swift

```diff
@@ -199,9 +177,20 @@ class HomeViewController: UIViewController {
     }
 
     @objc private func presentFeedback() {
```

Comment:
This function needs cleaning up. If you have a view that is sometimes nil, it should be optional `?` not `!`. The view should remove itself or tell its parent to remove it when feedback is finished. This function is called present feedback-- it doesn't make sense of it to only _remove_ the feedback view depending on when it was called.